invenio_drafts_resources/services/records/config.py

```diff
@@ -164,26 +167,33 @@ class RecordServiceConfig(RecordServiceConfigBase):
     links_item = {
         "self": ConditionalLink(
             cond=is_record,
-            if_=RecordLink("{+api}/records/{id}"),
-            else_=RecordLink("{+api}/records/{id}/draft"),
+            if_=RecordEndpointLink("records.read"),
```

Comment:
where is `records.read` defined? i understand `invenio_app_rdm_records`below is a blueprint, but i can't find a `records` blueprint

Reply:
is it [this](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/config.py#L239)?

but that would mean, we have the same problem as below with `invenio_app_rdm_records`, we introduce a implicit dependency to `invenio-rdm-records